otterai/main.py

```diff
@@ -125,12 +125,8 @@ def get_file_content(repo, file_path: str, commit_sha: str) -> str:
     except Exception:
         return ""
 
-def get_pr_diff(gh_token: str, repo_name: str, pr_number: int) -> List[Dict[str, Any]]:
+def get_pr_diff(repo: Repository.Repository, pr: PullRequest.PullRequest) -> List[Dict[str, Any]]:
```

Comment:
⚙️ Ensure that the `get_pr_diff` function handles potential exceptions when fetching the PR diff to avoid runtime errors. 🚨